openassessment/assessment/test/test_staff.py

```diff
@@ -196,6 +200,10 @@ def test_create_assessment_does_not_block(self, after_type, after_assess):
         if after_type == 'staff':
             return
 
+        requirements = self.STEP_REQUIREMENTS
```

Comment:
@efischer19 I'm having trouble understanding this test change-- can you explain?